Backend/Interview.Domain/Rooms/Service/RoomService.cs

```diff
@@ -139,6 +159,7 @@ public sealed class RoomService : IRoomServiceWithoutPermissionCheck
                     .ToList(),
                 RoomStatus = e.Status.EnumValue,
                 Tags = e.Tags.Select(t => new TagItem { Id = t.Id, Value = t.Value, HexValue = t.HexColor, }).ToList(),
+                Timer = mapTimer.Invoke(e.Timer),
```

Comment:
Because of this will pull all the necessary data first on the client, and then perform Select, it is better to just write the lambda code here with a ternary operator, then there will be no extra data upload and it will be 1 Select from the database.